WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -437,32 +461,6 @@ private extension SupportTableViewController {
         }
     }
 
-    struct ButtonRow: ImmuTableRow {
```

Comment:
Moving this out of this private extension to be able to reuse it from another place. Also renamed it as ExternalLinkButtonRow.